src/qibocal/fitting/classifier/ada_boost.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from sklearn.ensemble import AdaBoostClassifier
+from sklearn.model_selection import GridSearchCV, RepeatedStratifiedKFold
+
+from . import utils
+
+
+def hyperopt(x_train, y_train, _path):
+    r"""Perform an hyperparameter optimization and return the hyperparameters.
+
+    Args:
+        x_train: Training inputs.
+        y_train: Training outputs.
+        _path (path): Model save path.
+
+    Returns:
+        Dictionary with model's hyperparameters.
+    """
+    clf = AdaBoostClassifier()
+    cv = RepeatedStratifiedKFold(n_splits=10, n_repeats=3, random_state=1)
+    space = dict()
```

Comment:
```suggestion
    space = {}
```

The literal is considered more Pythonic.